code/game/objects/items/misc_items.dm

```diff
@@ -55,4 +55,19 @@
 /obj/item/red_ribbon_arm/unequipped(mob/living/carbon/human/user, var/from_slot = null)
 	..()
 	if(from_slot == slot_belt && istype(user))
-		user.set_hand_amount(user.held_items.len - 1)
\ No newline at end of file
+		user.set_hand_amount(user.held_items.len - 1)
+
+
+/obj/item/folded_bag
+	name = "folded plastic bag"
+	desc = "A neatly folded-up plastic bag, making it easier to store."
+	icon_state = "folded_bag"
+	w_class = W_CLASS_TINY
+
+/obj/item/folded_bag/attack_self(mob/user)
+	to_chat(user, "<span class = 'notice'>You unfold \the [src] into a plastic bag.</span>")
+	var/bag = new/obj/item/weapon/storage/bag/plasticbag
+	user.u_equip(src)
+	transfer_fingerprints_to(bag)
+	user.put_in_hands(bag)
```

Comment:
tl;dr of above
```
if(!user.put_in_hands(bag))
	bag.forceMove(user.loc)
```

Reply:
Why not just do `var/bag = new /obj/item/weapon/storage/bag/plasticbag(user.loc)`?
If the `put_in_hands()` succeeds it gets put in the user's hands. If it fails, the bag remains at the user's `loc`.